scripts/jenkins/ardana/ansible/cloud-ses-integration.yml

```diff
@@ -19,7 +19,7 @@
 - import_playbook: get-virt-ses-cluster.yml
   when: not is_physical_deploy
 
-- name: Configure SES for Ardana when SES enabled
+- name: Configure SES for Cloud
   hosts: "ses_{{ (ardana_env.startswith('pcloud') or ardana_env.startswith('qe')) | ternary('qe', 'virt') }}"
```

Comment:
should this remain as "ardana_env"?

Reply:
@dirkmueller replacing `ardana_env` with something like `cloud_env` is something we need to do globally. I have [this](https://jira.suse.de/browse/SCRD-8475) scheduled in the backlog, but its cosmetic work that has lower priority at this moment. We'll just have to live with it for a bit longer.